### PR TITLE
yq: 2.3.3 -> 2.3.4

### PR DESCRIPTION
**Pull request by mimame**
Update yq to the last version

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/tools/yq/default.nix [hunk 1]
```diff
@@ -4,7 +4,7 @@ buildPythonApplication rec {
 
   name = "${pname}-${version}";
```
Comment (dotlambda) [inline pkgs/development/tools/yq/default.nix]: Please get rid of the `name`?

Comment (mimadrid) [reply pkgs/development/tools/yq/default.nix]: Done

**Comment by andir** [comment]
@GrahamcOfBorg build yq

**Comment by andir** [comment]
@mimadrid thanks! 👍

**Pull request merged**